docs/visualization/plotting-nd-data.ipynb

```diff
@@ -74,7 +84,20 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "plot(d, axes=['z', 'Q_x', 'y', 'x'])"
+    "plot(d, axes={'x': 'x', 'y': 'Q_x'})"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Profile picking\n",
+    "\n",
+    "Finally, each dimension also comes with a `Profile` button which allows to display one of the additional dimensions as a profile underneath the main plot.\n",
+    "\n",
+    "- When hovering the mouse over the top image, the profile below is updated according to the mouse position.\n",
+    "- Clicking on the image will save the current profile with a random color.\n",
+    "- Clicking on an existing marker on the image will delete the corresponding saved profile."
```

Comment:
> > More of a discussion point than something obviously wrong - Additional dimension does not appear if one does:
> > `d.coords['xx'] = d.coords['x'].copy()` or similar and for good reason such as complexities around `rebin` . Explanation at top of notebook should probably be expanded to explicitly say that the chosen coordinates are dimension coordinates and non-dimension coordinates are not rendered.
> 
> I'm not sure how you would expect auxillary coordinates to be rendered? Have something like a double x axis?
> There are examples on how to use auxillary coords along axes by doing `plot(d, axes={'x': 'xx'})` which means "I want to show `xx` along the horizontal (x) axis of the plot".

Yes, I think that axes option is good and does what is needed.

Reply:
In trying this out, I did find that it did not use the values of the coordinates I had for say `xx` on the sliders. Try shifting xx and plotting.

Reply:
hmm i'm not sure i am supporting `xx` on sliders... don't think we ever did.
i think i need to update the slider readouts so that they use the axis formatters...

Reply:
i opened #1349